### PR TITLE
Corrected script for some item #2

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: N/A

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -90506,6 +90573,62 @@ Body:
...
+    Refineable: true
+    View: 219
+  - Id: 19909
+    AegisName: C_White_Deviruchi_Cap
+    Name: Costume White Baby demom hat
+    Type: Armor
+    Locations:
+      Costume_Head_Top: true
+    ArmorLevel: 1
+    Refineable: true
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:90630]: Thanks!

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -90506,6 +90573,62 @@ Body:
...
+    View: 710
+  - Id: 19908
+    AegisName: C_Joker_Jester_
+    Name: Costume National flag hat
+    Type: Armor
+    Locations:
+      Costume_Head_Top: true
+    ArmorLevel: 1
+    Refineable: true
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:90621]: Thanks!

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -90506,6 +90573,62 @@ Body:
...
+    View: 696
+  - Id: 19907
+    AegisName: C_Fancy_Phantom_Mask
+    Name: Costume Beautiful Phantom Mask
+    Type: Armor
+    Locations:
+      Costume_Head_Top: true
+    ArmorLevel: 1
+    Refineable: true
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:90612]: Thanks!

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -90506,6 +90573,62 @@ Body:
...
+    View: 806
+  - Id: 19906
+    AegisName: C_24_Bolt
+    Name: Costume 24 Bolt
+    Type: Armor
+    Locations:
+      Costume_Head_Top: true
+    ArmorLevel: 1
+    Refineable: true
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:90603]: Thanks!

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -90506,6 +90573,62 @@ Body:
...
+  - Id: 19905
+    AegisName: C_Secret_Society_Hood
+    Name: Costume Secret Bandana
+    Type: Armor
+    Locations:
+      Costume_Head_Top: true
+      Costume_Head_Mid: true
+      Costume_Head_Low: true
+    ArmorLevel: 1
+    Refineable: true
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:90594]: Thanks!

**Comment by Atemo** [comment]
Good work!

**Pull request merged**